tenders/hvt/hvt_openbsd_x86_64.c

```diff
@@ -186,9 +185,8 @@ int hvt_vcpu_loop(struct hvt *hvt)
                     /* Guest has halted the CPU. */
                     if (nr == HVT_HYPERCALL_HALT) {
                         hvt_gpa_t gpa = vei->vei.vei_data;
-                        struct hvt_halt *p =
-                            HVT_CHECKED_GPA_P(hvt, gpa, sizeof (struct hvt_halt));
-                        return p->exit_status;
+                        hvb->vrs = &vei->vrs;
```

Comment:
Any reason why this (or whatever call that fills `&vei`) can't be done in `hvt_dumpcore_write_prstatus()`?

Reply:
This can be done in `hvt_dumpcore_write_prstatus()` with another call back into the kernel. It is available here without that extra call, that is all.

I can change this to call back into kernel inside `hvt_dumpcore_write_prstatus()` if you would prefer?

Reply:
Yes please, I'd prefer the kernel call local to its usage. Thanks.